bitcoind-regtest/bitcoind-regtest.cabal

```diff
@@ -25,8 +25,8 @@ common core
         , haskoin-core >=0.15 && <0.22
         , http-client >=0.6 && <0.8
         , process ^>=1.6
-        , servant >=0.19 && <0.20
-        , servant-client >=0.19 && <0.20
+        , servant >=0.19 && <0.21
+        , servant-client >=0.19 && <0.21
```

Comment:
Should the lower bound be `0.19.1`?

Reply:
I have restored the pinning on `servant-client-0.19.1` upping the lower bounds did not work because `0.19.1` is not a published version.

Reply:
Ok re-removed the pin, as we agreed offline, we would let downstream users deal with it if they need to work with `0.19.*`.